relay-server/src/actors/store.rs

```diff
@@ -317,19 +317,16 @@ impl StoreForwarder {
     ) -> Result<(), StoreError> {
         let payload = item.payload();
 
-        // NB: Metrics are guaranteed to contain a timestamp at this point.
-        for metric_result in Metric::parse_all(&payload, UnixTimestamp::from_secs(0)) {
-            if let Ok(metric) = metric_result {
-                self.send_metric_message(MetricKafkaMessage {
-                    org_id,
-                    project_id,
-                    name: metric.name,
-                    unit: metric.unit,
-                    value: metric.value,
-                    timestamp: metric.timestamp,
-                    tags: metric.tags,
-                })?;
-            }
+        for bucket in Bucket::parse_all(&payload).unwrap_or_default() {
```

Comment:
@jan-auer It feels wrong that we have to deserialize here. Is there any way we can bypass the serialization step?

Reply:
Agreed, and we have the same issue with sessions. Right now, there's no straight forward way to avoid it, since we have to transform the payload and insert new attributes. 

We could move this step into the EventManager, bypassing the Envelope, but I'm not clear yet what a good solution would look like. For now, I'd propose to keep it this way and solve both metrics and sessions together.

Reply:
Sounds good.